eth_typing/abi.py

```diff
@@ -64,7 +63,7 @@ class ABIComponent(TypedDict):
     """Name of the component."""
     type: str
     """Type of the component."""
-    components: NotRequired[Sequence["ABIComponent"]]
+    components: Sequence["ABIComponent"]
```

Comment:
Does ABIComponent need a `total=False` too?

Reply:
That got fuzzy to trace back, good callout since I broke this in #74 but hasnt yet been released.